apps/web/src/views/TradingReward/hooks/useAllUserCampaignInfo.tsx

```diff
@@ -136,8 +136,9 @@ const useAllUserCampaignInfo = ({ campaignIds, type }: UseAllUserCampaignInfoPro
       }
     },
     {
-      refreshInterval: SLOW_INTERVAL,
-      fallbackData: [],
+      refetchInterval: SLOW_INTERVAL,
+      initialData: [],
+      enabled: Boolean(campaignIds.length > 0 && account),
```

Comment:
There's a missing `type`?
```suggestion
      enabled: Boolean(campaignIds.length > 0 && account && type),
```